config/config.go

```diff
@@ -37,6 +38,10 @@ type EnclaveConfig struct {
 	AttestationProviderAddr string `mapstructure:"attestation-provider-addr"`
 }
 
+type EndPointConfig struct {
+	PublicEndPointURL string `mapstructure:"public-endpoint-url"`
+}
+
```

Comment:
I think `PublicEndPointURL` is the setting of this Validator, so it seems better to declare `BaseConfig`

Reply:
Oh it's seems to be good. I'll change it at [this branch](https://github.com/medibloc/panacea-data-market-validator/pull/68).